plugins/woocommerce/client/legacy/js/frontend/a8c-address-autocomplete-service.js

```diff
@@ -0,0 +1,485 @@
+( function () {
+	const permanentlyDisabledServices = [];
```

Comment:
I keep count of services disabled here, just in case we have a broken thing, it just auto skip itself in canSearch

Reply:
Disabling is per client session, if you refresh the page, the limits are gone.